### DISCUSSION TITLE
Minor Keys in AlphaTex

### DISCUSSION THREAD

**Discussion by jonaro00** [CoderLine/alphaTab#853] [Ideas]
## Proposed feature
AlphaTex currently allows defining major key signatures with `\ks G`. I propose also allowing minor keys by simply expanding the switch-cases in `AlphaTexImporter.parseKeySignature`.

What the syntax would be is up for discussion. My idea is to keep the default behavior for major keys, using `\ks Eminor` for minor, but also allowing `\ks Gmajor` for consistency.

The switch-case would then look like:
```
...
case 'g':
case 'gmajor':
case 'eminor':
    ...
```

**Comment by Danielku15** [comment]
Sounds like a good extension to me. Syntax is also good. Added https://github.com/CoderLine/alphaTab/issues/875
